guides/v2.2/frontend-dev-guide/css-guide/css_quick_guide_mode.md

```diff
@@ -144,6 +144,8 @@ If you are using server-side compilation mode, you must [clean generated static
 1. Refresh the page and verify your changes.
 
     ![Admin login page where the font of the buttons was changed]({{ site.baseurl }}/common/images/extend_less_screenshot221.png)
+    
+1. Always use variables instead of hardcoded values like `color: red`
```

Comment:
When we view your change in the context of the list of steps it doesn't seem to flow. Let's change this to a tip:
{:.bs-callout .bs-callout-tip}
Always use variables instead of hardcoded values like `color: red`

Reply:
@eriksmarr I have updated change as per suggestions, please review